### PR TITLE
feat: Sort results alphabetically by prefLabel

### LINKED ISSUE #68: Sort results alphabetically
This is to prevent questions about the order that search results appear in. Our SPARQL `CONSTRUCT` queries preclude us from keeping the source’s ordering or providing our own `ORDER BY`. The sort should therefore happen on the client-side (resolvers?).

### PR DESCRIPTION
**Pull request by ddeboer**
Fix #68.

### CODE REVIEW

File: src/services/query.ts [hunk 1]
```diff
@@ -106,3 +108,17 @@ export class QueryTermsService {
...
+  const prefLabelB = b.prefLabels[0]?.value;
+  if (prefLabelA < prefLabelB) {
+    return -1;
+  }
+
+  if (prefLabelA > prefLabelB) {
+    return 1;
+  }
+
+  return 0;
```
Comment (sdevalk) [inline src/services/query.ts:123]: Or we can use `return prefLabelA.localeCompare(prefLabelB)` - a bit more concise?

**Review by sdevalk — CHANGES_REQUESTED** [review]

File: src/services/query.ts (hunk 1 above)
Comment (ddeboer) [reply src/services/query.ts]: Nice! Changed.

File: src/services/query.ts [hunk 2]
```diff
@@ -106,3 +108,9 @@ export class QueryTermsService {
     });
   }
 }
+
+const alphabeticallyByPrefLabel = (a: Term, b: Term) => {
+  const prefLabelA = a.prefLabels[0]?.value ?? '';
+  const prefLabelB = b.prefLabels[0]?.value ?? '';
```
Comment (ddeboer) [inline src/services/query.ts:114]: `localCompare` must take a non-null string, so default to an empty string here.

Comment (sdevalk) [reply src/services/query.ts:114]: 👍🏻

**Review by sdevalk — APPROVED** [review]

**Pull request merged**